lib/src/app.dart

```diff
@@ -1,25 +1,41 @@
 import 'package:flutter/material.dart';
-
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:http/http.dart' as http;
+import '/../src/services/detail_service/app_detail_service.dart';
```

Comment:
insert a line break

Reply:
Does your mind insert a line break between "package in flutter" and  "package project"?